### PR TITLE
Style CommentUpdateForm

### PR DESCRIPTION
**Pull request by estherchee**
- Initiate useState of length so characters display on CommentUpdateForm reflect the number of characters of the original comment
- Set the margin of .length css to zero to close the height gap between characters display and button
- Style CommentUpdateForm

Technical Requirement:
- Style CommentUpdateForm

Screenshot:
Screenshot 2021-05-12 at 00 33 53

### CODE REVIEW

File: frontend/src/styles/components/commentform.css [hunk 1]
```diff
@@ -17,6 +17,7 @@
 }
 
 .length {
```
Comment (NeuralAlchemist) [inline frontend/src/styles/components/commentform.css:19]: i can't tell but is there extra space after the dot? is the styling applied? if not please remove the space

File: frontend/src/styles/components/commentupdateform.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+.comment {
+  margin-right: var(--space-xx-small);
+  height: var(--space-medium);
+}
+
+.icon-container {
+  display: flex;
+}
+
+.length {
```
Comment (NeuralAlchemist) [inline frontend/src/styles/components/commentupdateform.css:19]: i can't tell but is there extra space after the dot? is the styling applied? if not please remove the space

**Review by NeuralAlchemist — COMMENTED** [review]
This is maybe not needed right now but could the update comment button be on the same line as the character limit display? Please turn this into enchancement issue if you think its not important.

File: frontend/src/styles/components/commentform.css (hunk 1 above)
Comment (eeels22) [reply frontend/src/styles/components/commentform.css:19]: I don't think there's a space

File: frontend/src/styles/components/commentupdateform.css (hunk 2 above)
Comment (eeels22) [reply frontend/src/styles/components/commentupdateform.css:19]: I don't think there's a space

**Review by NeuralAlchemist — APPROVED** [review]
looks good!

**Review by janko101 — APPROVED** [review]

**Review by eeels22 — APPROVED** [review]
Thanks Pei!

**Pull request merged**